app/src/org/odk/collect/android/views/media/MediaLayout.java

```diff
@@ -153,6 +153,9 @@ public void onClick(View v) {
             videoParams.addRule(RelativeLayout.BELOW, mAudioButton.getId());
             topPane.addView(mAudioButton, audioParams);
             topPane.addView(mVideoButton, videoParams);
+        } else {
+            //Audio and Video are both null, let text bleed to right
+            textParams.addRule(RelativeLayout.ALIGN_PARENT_RIGHT);
```

Comment:
@ctsims This was included in commit 2902bf4ddbbf9e4194dd960291ddd923d97dd1b7
 "Temporary fix for markdown which retains the video/audio playback, but also doesn't require trailing spaces."

Should I revert ?